### ISSUE TITLE
Loglike is actually the squared z-score

### ISSUE THREAD

**Issue by LouisDesdoigts** [benjaminpope/drpangloss#26]
The [`loglike` function](https://github.com/benjaminpope/drpangloss/blob/main/src/drpangloss/models.py#L390) isn't actually the log likelihood, but the z-score. This is fine for optimization and monochromatic analysis, but will give incorrect results when trying to combine likelihoods across multiple bands.

Simple fix here:

```python
from jax.scipy.stats import norm

def loglike(values, params, data_obj, model_class):
    data, errors = data_obj.flatten_data()
    param_dict = dict(zip(params, values))
    model_data = data_obj.model(model_class(**param_dict))
    return norm.logpdf(model_data, loc=data, scale=errors).sum()
```

I can create a PR on this, so this is more a reminder than anything.

**Comment by benjaminpope** [comment]
yep, just need to normalize this properly
